deployment/gke/README.md

```diff
@@ -0,0 +1,24 @@
+# Deployt website to multi GKE cluster
+
+## Prerequisites
+
+- Create global static IP in GCP.
+- Configure the DNS in the domain registra.
+- [Grant the required IAM roles](https://cloud.google.com/anthos/multicluster-management/connect/prerequisites#grant_iam_roles).
+
+## One Time Setup
+
+```bash
+../generate_yaml.sh <staging | prod>
+onetime_setup.sh
+```
+
+This step creates clusters and make all the one time setup, including:
```

Comment:
"make all the one time setup" is a bit confusing:

```suggestion
This step creates clusters and runs the one time setup tasks, including:
```

Reply:
Done